lib/linked-list.js

```diff
@@ -13,105 +13,207 @@ class LinkedList {
     constructor() {
         // The # before the variable name indicates
         //   a private variable.
-        this.#head = null;
+        this.head = null;
+        this.tail = null;
+        this.size = 0;
     }
 
     /* 
     Method to retrieve the value in the first node.
     returns null if the list is empty.
-    Time Complexity: ?
-    Space Complexity: ?
+    Time Complexity: O(1)
+    Space Complexity: O(n)
     */
     getFirst() {
```

Comment:
Shouldn't this method also check to see if the node is empty and return the value of the node?